src/hw/pcie_cfg.cpp

```diff
@@ -97,32 +139,16 @@ bool pcieCfg::addBifurcationConfig(const int& socket,
         {
             return true;
         }
-        else if (((old_value == pcieBifurcateX4X4XXX8) ||
-                  (old_value == pcieBifurcateXXX8X4X4)) &&
-                 (value == pcieBifurcateXXX8XXX8))
-        {
-            // We want to set our half to 'x8', while other component split it's
-            // half into 'x4x4' - keep old value, since it has already set 'x8'
-            // for us
-            return true;
-        }
-        else if ((old_value == pcieBifurcateXXX8XXX8) &&
-                 ((value == pcieBifurcateX4X4XXX8) ||
-                  (value == pcieBifurcateXXX8X4X4)))
+        else if (((old_value & 0x0f) == 0x0f) && ((value & 0xf0) == 0xf0))
         {
-            // We want to split our half into 'x4x4', while other component uses
-            // it's half as 'x8' - set our value
-            bifurcationConfig[instance] = value;
+            // merge High half of old value with Low half of current value
+            bifurcationConfig[instance] = (old_value & 0xf0) | (value & 0x0f);
```

Comment:
What if the new value is `bmcPcieBifurcateXXXXXXXX` that is `0xFF` ? Will that work?

Reply:
Right, this is special case and should be handled separately. Will fix.